boards/common/esp32/Makefile.include

```diff
@@ -8,4 +8,10 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
 include $(RIOTMAKE)/tools/serial.inc.mk
 
 # reset tool configuration
-export RESET = esptool.py --before default_reset run
+ifeq (1,$(ESP_RESET_DTRTOGGLE))
+  # use dtr tool from https://github.com/kaspar030/rtsdtr for reset
+  # used by Murdock
+  export RESET = dtr $(PORT) 0 && dtr $(PORT) 1
+else
+  export RESET ?= esptool.py --before default_reset run
```

Comment:
Unfortunately not, it doesn't work with the "make test" target. In that use-case, the test terminal program has the tty open, then "make reset" is called. esptool tries to connect to the bootloader instead of just resetting via RTS/DTR, and fails because the test terminal is hijacking the communication.

Also, on the RPi workers, spawning python takes seconds. ;)

Reply:
Is there not a python alternative for RIOT that could be included in the repo ?

I mean, flashing is done with python, the tests are run with python, so using python for this would be easier than having to install external programs.

And the RPI could use its own C implementation anyway by overwriting `RESET` and `RESET_FLAGS`.

Reply:
Ideally local testing is not much different from what murdock does. And for Murdock, spawning python is *slow*. We should maybe just integrate the (5 line) c tool into the RIOT repo.